chromaui/chromatic-e2e
# Fix ESM build

Issue: #

## What Changed
Fixes #88 

Fix ESM build by:
* Replacing `fs-extra` with standard `fs` operations.
* Fixing use of `require.resolve`.



## How to test
* See all still works
* Try out canary on an ESM project (esp cypress) (`"type": "module"` in `package.json`)
* See this [passing playwright build using the canary](https://github.com/chromaui-demo/e2e-demo-playwright/actions/runs/8252368610/job/22571695133)
* See this [passing cypress build using the canary](https://github.com/chromaui-demo/e2e-demo-cypress/actions/runs/8254589076/job/22579079410)
